scripts/process_data_dcm-zurich.sh

```diff
@@ -169,9 +211,11 @@ else
     fi
 
     # Label T2w axial spinal cord segmentation. Either using manual disc labels or using disc labels from sagittal image.
-    # Note: here we use sct_label_utils instead of sct_label_vertebrae to avoid SC straightening
+    # Before:"# Note: here we use sct_label_utils instead of sct_label_vertebrae to avoid SC straightening"
+    #sct_label_utils -i ${file_t2_ax_seg}.nii.gz -disc ${file_t2_ax_labels}.nii.gz -o ${file_t2_ax_seg}_labeled.nii.gz
+    # Note: Updated to use sct_label_vertebrae -discfile instead of deprecated sct_label_utils -disc
     # Context: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4072
-    sct_label_utils -i ${file_t2_ax_seg}.nii.gz -disc ${file_t2_ax_labels}.nii.gz -o ${file_t2_ax_seg}_labeled.nii.gz
+    sct_label_vertebrae -i ${file_t2_ax}.nii.gz -s ${file_t2_ax_seg}.nii.gz -discfile ${file_t2_ax_labels}.nii.gz -c t2
```

Comment:
Migth also be worth testing using [totalspineseg](https://spinalcordtoolbox.com/stable/user_section/command-line/deepseg/totalspineseg.html) directly on the axial image and compare results!

Reply:
comparing getting the vertebral labels from the sagittal image vs getting it using totalspineseg directly on the axial, and doing a QC report comparing the quality of the labeling, is it more clear?